source/hpack.c

```diff
@@ -1261,9 +1261,7 @@ int aws_hpack_decode(
                 goto handle_complete;
             } break;
 
-            default: {
```

Comment:
When I run Format Document at windows machine, this will happen. Undo this change will pass the Lint for format...

Reply:
Yeah, for some reason, the clang format on my machine is slightly out of sync with what's in CI